### PR TITLE
Improve stubs in av.video

### PR DESCRIPTION
**Pull request by laggykiller**

### CODE REVIEW

File: av/video/stream.pyi [hunk 1]
```diff
@@ -4,16 +4,16 @@ from typing import Any
...
 
 class VideoStream(Stream):
-    name: str
     width: int
     height: int
+    format: VideoFormat
     pix_fmt: str | None
     sample_aspect_ratio: Fraction | None
-    codec_context: VideoCodecContext
+    type = "video"
```
Comment (WyattBlue) [inline av/video/stream.pyi:16]: Should be a Literal
codec_context in VideoStream is always VideoCodecContext so I think we should preserve that type

File: av/video/frame.pyi [hunk 2]
```diff
@@ -19,34 +28,37 @@ class PictureType(EnumItem):
 
 class VideoFrame(Frame):
     format: VideoFormat
-    pts: int
-    time: float
```
Comment (WyattBlue) [inline av/video/frame.pyi:23]: This should remain because pts and time are always not None in VideoFrame

**Comment by laggykiller** [comment]
I wanted to also add `codec_context` typing in av.audio.stream.AudioStream, but that caused conflict...

**Comment by WyattBlue** [comment]
You should just overwrite your previous commit then force push. It's okay when it's on your branch.

**Comment by laggykiller** [comment]
Should I open another PR for changing `codec_context` in `av.audio.stream.AudioStream`? Or should I just leave that to you?

File: av/video/stream.pyi (hunk 1 above)
Comment (laggykiller) [reply av/video/stream.pyi]: Thanks, resolved!

File: av/video/frame.pyi (hunk 2 above)
Comment (laggykiller) [reply av/video/frame.pyi]: Thanks, resolved!

File: av/video/frame.pyi [hunk 3]
```diff
@@ -19,34 +28,37 @@ class PictureType(EnumItem):
...
     planes: tuple[VideoPlane, ...]
     width: int
     height: int
     key_frame: bool
     interlaced_frame: bool
     pict_type: int
+    colorspace: Colorspace
+    color_range: ColorRange
```
[the remark this answers is not in the document]
Comment (laggykiller) [reply av/video/frame.pyi]: Thanks, resolved!

**Comment by WyattBlue** [comment]
Once a PR is merged in `main`. You can't revert it. 

Just make a new PR.

**Comment by laggykiller** [comment]
Sure! Opened #1321

**Pull request merged**